.github/workflows/osx.yml

```diff
@@ -0,0 +1,43 @@
+---
+name: macOS10.14 build and test
+
+on:
+  pull_request:
+    branches:
+      - master
+
+
+jobs:
+  OSX:
+    runs-on: macOS-10.14
+
+    steps:
+      - name: Install Dependencies
```

Comment:
FYI, `name` isn't required for steps; if you don't supply a name, GA just uses the name of the published action or the command (as appropriate for the step). Personally I like that better for simple steps like this, but YMMV.